FetchMigration/README.md

```diff
@@ -1,43 +1,38 @@
-# Index Configuration Tool
+# "Fetch" Data Migration / Backfill
 
-Python package that automates the creation of indices on a target cluster based on the contents of a source cluster. 
-Index settings and index mappings are correctly copied over, but no data is transferred. 
-This tool seeks to eliminate the need to [specify index templates](https://github.com/awslabs/logstash-output-amazon_es#optional-parameters) when migrating data from one cluster to another.
-The tool currently supports ElasticSearch or OpenSearch as source and target.
+Fetch Migration provides an easy-to-use tool that simplifies the process of moving indices and their data from a 
+"source" cluster (either Elasticsearch or OpenSearch) to a "target" OpenSearch cluster. It automates the process of 
+comparing indices between the two clusters and only creates index metadata (settings and mappings) that do not already 
+exist on the target cluster. Internally, the tool uses [Data Prepper](https://github.com/opensearch-project/data-prepper) 
+to migrate data for these created indices.
 
-## Parameters
+The Fetch Migration tool is implemented in Python.
+A Docker image can be built using the included [Dockerfile](./Dockerfile).
 
-The first required input to the tool is a path to a [Data Prepper](https://github.com/opensearch-project/data-prepper) pipeline YAML file, which is parsed to obtain the source and target cluster endpoints.
-The second required input is an output path to which a modified version of the pipeline YAML file is written.
-This version of the pipeline adds an index inclusion configuration to the sink, specifying only those indices that were created by the index configuration tool.
-The tool also supports several optional flags:
+## Components
 
-| Flag | Purpose |
-| ------------- | ------------- |
-| `-h, --help`    | Prints help text and exits |
-| `--report, -r`  | Prints a report of indices indicating which ones will be created, along with indices that are identical or have conflicting settings/mappings.  |
-| `--dryrun`      | Skips the actual creation of indices on the target cluster |
+The tool consists of 3 components:
+* A "metadata migration" module that handles metadata comparison between the source and target clusters. 
+This can output a human-readable report as well as a Data Prepper pipeline `yaml` file.
+* A "migration monitor" module that monitors the progress of the migration and shuts down the Data Prepper pipeline 
+once the target document count has been reached
+* An "orchestrator" module that sequences these steps as a workflow and manages the kick-off of the Data Prepper 
+process between them.
 
-### Reporting
-
-If `--report` is specified, the tool prints all processed indices organized into 3 buckets:
```

Comment:
Is this no longer the case?

Reply:
Yes. It is turned on by default in `fetch_orchestrator.py` to allow for meaningful human-readable log output:

https://github.com/kartg/opensearch-migrations/blob/e9501a071c4bf7be7006d8e91e522d57b65ae920/FetchMigration/python/fetch_orchestrator.py#L20